templates/admin/nav.tmpl

```diff
@@ -1,16 +0,0 @@
-<div id="setting-menu" class="grid-1-5 panel panel-radius left">
```

Comment:
Why this file was deleted?

Reply:
I couldn't find any references to it in the code base, and no changes other than 1 for the last 4 years. I originally modified it before I realised `navbar` was the right one.

Reply:
Please open sepate PR for that